src/OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -114,6 +118,15 @@ constexpr const char* kLogTimeFormat = "%Y-%m-%dT%H:%M:%E6S";
   }
 
 extern std::ofstream log_file;
+[[nodiscard]] std::vector<std::filesystem::path> ListFiles(const std::filesystem::path& log_dir);
+ErrorMessageOr<absl::Duration> GetLogFileLifetime(const std::string& log_file_name);
+[[nodiscard]] std::vector<std::filesystem::path> FilterOldLogFiles(
+    const std::vector<std::filesystem::path>& log_file_paths);
+ErrorMessageOr<void> RemoveFiles(const std::vector<std::filesystem::path>& log_file_paths);
```

Comment:
You can move them to src/OrbitBase/ and create separate header file. This will make them private to OrbitBase component but you still can use it from the test.

Reply:
Thanks so much for telling me this! I moved these functions to `LoggingUtils.h` and `LoggingUtils.cpp`.